cql3/Cql.g

```diff
@@ -153,30 +153,11 @@ struct uninitialized {
 
     bind_variable new_bind_variables(shared_ptr<cql3::column_identifier> name)
     {
-        auto marker = bind_variable{bind_variable::shape_type::scalar, _bind_variables.size()};
+        auto marker = bind_variable{_bind_variables.size()};
```

Comment:
Again, for people less experienced in this code it would be good to expand a little bit on the commit message - what was this "shape_type" thing used for in the past, and why is it no longer needed?